smart-contracts/contracts/mixins/MixinDisableAndDestroy.sol

```diff
@@ -58,8 +59,13 @@ contract MixinDisableAndDestroy is
     onlyOwner
   {
     require(isAlive == false, 'DISABLE_FIRST');
+
     emit Destroy(address(this).balance, msg.sender);
+
+    // this will send any ETH or ERC20 held by the lock to the owner
+    _transfer(msg.sender, _getBalance(address(this)));
```

Comment:
does `transfer` automatically just send whatever tokens the contract owns? Just wondering how it knows to send ETH or erc20 ?

Reply:
It uses the Funds mixin, which has the tokenAddress stored from when the token was created.  So if it's an ERC20 lock this will send ERC20 tokens.  Then selfdestruct will send any ETH if somehow the contract acquired some.

Reply:
Ah, of course. sounds good.